server/src/main/java/com/scalar/db/server/ScalarDbServer.java

```diff
@@ -58,16 +59,23 @@ public void start() throws IOException {
     ServerConfig config = new ServerConfig(properties);
     Injector injector =
         Guice.createInjector(new ServerModule(config, new DatabaseConfig(properties)));
-    server =
+
+    ServerBuilder<?> builder =
         ServerBuilder.forPort(config.getPort())
             .addService(injector.getInstance(DistributedStorageService.class))
             .addService(injector.getInstance(DistributedStorageAdminService.class))
             .addService(injector.getInstance(DistributedTransactionService.class))
             .addService(injector.getInstance(AdminService.class))
             .addService(new HealthService())
-            .addService(ProtoReflectionService.newInstance())
-            .build()
-            .start();
+            .addService(ProtoReflectionService.newInstance());
+
+    // Two-phase commit for JDBC is not supported for now
+    String transactionManager = properties.getProperty(DatabaseConfig.TRANSACTION_MANAGER);
+    if (Strings.isNullOrEmpty(transactionManager) || !transactionManager.equals("jdbc")) {
```

Comment:
What would happen if `jdbc` is specified for the transaction manager?
I think it should throw an exception but does it?

Reply:
In the current implementation, `TwoPhaseCommitTransactionService` doesn't start.

Reply:
Just a confirmation. It doesn't start and throws some exception?

Reply:
It doesn't throw any exception when `jdbc` is specified for the transaction manager. Other services (DistributedStorageService, DistributedStorageAdminService, DistributedTransactionService) start in that case.